src/VCR_InterfaceTasks.cpp

```diff
@@ -15,52 +15,102 @@
 #include "VehicleStateMachine.h"
 #include "AMSSystem.h"
 
+
+bool init_bundle(){
+
+    float adc0_scales[channels_within_mcp_adc], adc0_offsets[channels_within_mcp_adc], adc1_scales[channels_within_mcp_adc], adc1_offsets[channels_within_mcp_adc]; 
+    adc0_scales[GLV_SENSE_CHANNEL] = GLV_SENSE_SCALE;
+    adc0_offsets[GLV_SENSE_CHANNEL] = GLV_SENSE_OFFSET;
+    adc0_scales[CURRENT_SENSE_CHANNEL] = CURRENT_SENSE_SCALE;
+    adc0_offsets[CURRENT_SENSE_CHANNEL] = CURRENT_SENSE_OFFSET;
+    adc0_scales[REFERENCE_SENSE_CHANNEL] = REFERENCE_SENSE_SCALE;
+    adc0_offsets[REFERENCE_SENSE_CHANNEL] = REFERENCE_SENSE_OFFSET;
+    adc0_scales[RL_LOADCELL_CHANNEL] = RL_LOADCELL_SCALE;
+    adc0_offsets[RL_LOADCELL_CHANNEL] = RL_LOADCELL_OFFSET;
+    adc0_scales[RR_LOADCELL_CHANNEL] = RR_LOADCELL_SCALE;
+    adc0_offsets[RR_LOADCELL_CHANNEL] = RR_LOADCELL_OFFSET;
+    adc0_scales[RL_SUS_POT_CHANNEL] = RL_SUS_POT_SCALE;
+    adc0_offsets[RL_SUS_POT_CHANNEL] = RL_SUS_POT_OFFSET;
+    adc0_scales[RR_SUS_POT_CHANNEL] = RR_SUS_POT_SCALE;
+    adc0_offsets[RR_SUS_POT_CHANNEL] = RR_SUS_POT_OFFSET;
+
+    adc1_scales[THERMISTOR_0] = THERMISTOR_0_SCALE;
+    adc1_offsets[THERMISTOR_0] = THERMISTOR_0_OFFSET;
+    adc1_scales[THERMISTOR_1] = THERMISTOR_1_SCALE;
+    adc1_offsets[THERMISTOR_1] = THERMISTOR_1_OFFSET;
+    adc1_scales[THERMISTOR_2] = THERMISTOR_2_SCALE;
+    adc1_offsets[THERMISTOR_2] = THERMISTOR_2_OFFSET;
+    adc1_scales[THERMISTOR_3] = THERMISTOR_3_SCALE;
+    adc1_offsets[THERMISTOR_3] = THERMISTOR_3_OFFSET;
+    adc1_scales[THERMISTOR_4] = THERMISTOR_4_SCALE;
+    adc1_offsets[THERMISTOR_4] = THERMISTOR_4_OFFSET;
+    adc1_scales[THERMISTOR_5] = THERMISTOR_5_SCALE;
+    adc1_offsets[THERMISTOR_5] = THERMISTOR_5_OFFSET;
+    adc1_scales[THERMISTOR_6] = THERMISTOR_6_SCALE;
+    adc1_offsets[THERMISTOR_6] = THERMISTOR_6_OFFSET;
+    adc1_scales[THERMISTOR_7] = THERMISTOR_7_SCALE;
+    adc1_offsets[THERMISTOR_7] = THERMISTOR_7_OFFSET;
+
+    ADCSingletonInstance::create(adc0_scales, adc0_offsets, adc1_scales, adc1_offsets);
+
+    return true;
+}
+
+
 bool init_read_adc0_task()
```

Comment:
You shouldn't need two init functions. The printf() and "return true" aren't important (they were just there for testing).

Reply:
You can just call init_bundle() in the init() function of main.cpp